libs/ballot-interpreter-nh/src/timing_marks.rs

```diff
@@ -403,15 +403,14 @@ pub fn find_timing_mark_shapes(
     let candidate_timing_marks = contours
         .iter()
         .enumerate()
-        .filter_map(|(i, contour)| {
+        .filter_map(|(_i, contour)| {
```

Comment:
Just `_` is the idiomatic way to do it if you want to ignore a parameter.

Reply:
👍 good to know. I was just following the compiler suggestion 🤷

Reply:
Also works. In practice I most see just underscore.